app/templates/admin/createEvent.html

```diff
@@ -20,20 +20,33 @@
 {% set isNewEvent = 'create' in request.path %}
 <!-- Program manager isCeltsAdmin and they are a manager for the program  -->
 <div class="text-center mb-3">
-    {% if isNewEvent %}
-      {% if template.tag == 'single-program' %}
-        {% set page_title = 'Create Event for ' + eventData["program"].programName %}
-      {% elif template.tag == 'all-volunteer' %}
-        {% set page_title = 'Create All Volunteer Training' %}
-      {% elif template.tag == 'no-program' %}
-        {% set page_title = 'Create Other CELTS-Sponsored Event' %}
-      {% else %}
-        {% set page_title = 'Create ' + template.name + ' Event' %}
-      {% endif %}
+  {% if isNewEvent %}
+    {% if template.tag == 'single-program' %}
+      {% set page_title = 'Create Event for ' + eventData["program"].programName %}
+    {% elif template.tag == 'all-volunteer' %}
+      {% set page_title = 'Create All Volunteer Training' %}
+    {% elif template.tag == 'no-program' %}
+      {% set page_title = 'Create Other CELTS-Sponsored Event' %}
+    {% else %}
+      {% set page_title = 'Create ' + template.name + ' Event' %}
+    {% endif %}
+    {% else %}
```

Comment:
I would unindent the else here as it currently looks like it is with the other conditional block which could be confusing.

Reply:
I would also fix the indentation for all the code under it to clear up any confusion.

Reply:
fixed indentation